### DISCUSSION TITLE
starting a background service worker

### DISCUSSION THREAD

**Discussion by chriskduffy** [coollabsio/coolify#4593] [Q&A]
Hey, i have my nextjs app running under nixpacks just fine on v4, beta.373. 

I want to run a background worker to generate new versions of pdfs that the users will create / edit. I am using this command locally: `npx tsx src/worker/generatePDFWorker.ts` from a new terminal shell in the root of my project. 

This works well in dev mode however when trying to figure out how to add a coolify resource / app, i've hit a brick wall. I've tried looking at Docker File as this doesn't need any ports open unlike if i try and use the same git repo i've deployed from...however just couldn't get it to see that file...it was saying something along the lines of path doesn't exist et {uuid}/src

Is it possibly something to do with Connect to Predefined Networks?

Basically just wanted to see if anyone out there has had any success setting up background process in coolify!

Thanks in advance!
Chris
